hw/ip/otbn/dv/rig/rig/gens/untaken_branch.py

```diff
@@ -0,0 +1,94 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+import random
+from typing import Optional
+
+from shared.insn_yaml import InsnsFile
+
+from ..config import Config
+from ..model import Model
+from ..program import ProgInsn, Program
+
+from ..snippet import ProgSnippet
+from ..snippet_gen import GenCont, GenRet
+from .branch_gen import BranchGen
+
+
+class UntakenBranch(BranchGen):
+    '''A snippet generator to generate aligned or
+       unalligned addresses for which branches are not taken.
```

Comment:
Nit: Unaligned (one L).

Also (if you're touching it anyway), the usual convention is to have a single line "title" and then paragraphs separated by blank lines. Here, I wouldn't bother talking about alignment; I'd just say something like "A snippet generator for branches that are not taken".

Reply:
Changed comment